src/Components/Todos.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import NewTodo from "./NewTodo";
+import Todo from "./Todo";
+import { useSelector } from "react-redux";
+
+export default function Todos() {
+  const todos = useSelector((state) => state.todos);
+
+  const [showNewTodoForm, setShowNewTodoForm] = useState(false);
+
+  const handleCreate = () => {
+    setShowNewTodoForm(true);
+  };
+
+  const handleNewSave = () => {
+    setShowNewTodoForm(false);
+  };
+
+  return (
+    <div>
+      <h1>Add Tasks</h1>
+      <button onClick={handleCreate}>Create</button>
+      <br />
```

Comment:
May be we can avoid using `<br />` here. I am adding a [reference](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br) which discussed good a best practice regarding its use cases. As it's not an official reference may be you can discuss it with mentor. Thanks

Reply:
Thanks for sharing, addressed.